ReSharper.FSharp/src/FSharp/FSharp.Psi.Daemon/src/Stages/PatternTypeHintsStage.fs

```diff
@@ -21,28 +22,39 @@ open JetBrains.ReSharper.Plugins.FSharp.Psi.Services.Util.TypeAnnotationsUtil
 open JetBrains.TextControl.DocumentMarkup.Adornments.IntraTextAdornments
 
 type private NodesRequiringHints =
-    { TopLevelNodes: VisibilityConsumer<ITreeNode>; LocalNodes: VisibilityConsumer<ITreeNode> } with
+    { TopLevelNodes: VisibilityConsumer<ITreeNode>
+      LocalNodes: VisibilityConsumer<ITreeNode>
+      OtherPatterns: VisibilityConsumer<ITreeNode> } with
```

Comment:
This naming seems inconsistent a bit. Shouldn't the nodes from `OtherPatterns` be the subset of `LocalNodes`, according to this naming? Or are `LocalNodes` about binding patterns specifically?